### PR TITLE
Fix server shutdown

### LINKED ISSUE #9367: Correctly stop Electron Hosted Instance of Theia
Related to Theia Plugin Model branch.

### Description
When user runs plugin for development in hosted mode using Theia Electron packaging, sometimes  Electron instance is not stopped correctly, so a process, which holds a port, remains.

### Reproduction Steps
1. Run Hosted Instance of Theia in Electron mode.
2. Close Electron window.
3. Terminate Hosted Instance.

Note, if swap steps 2 and 3, then the resources will be released properly.

### PR DESCRIPTION
**Pull request by mmorhun**
Signed-off-by: Mykola Morhun <mmorhun@redhat.com>

This PR fixes Theia server shutdown problem in case of electron and backend in docker container.
This PR introduces new external dependency: [ps-tree](https://www.npmjs.com/package/ps-tree)
Fixes: https://github.com/eclipse/che/issues/9367

### CODE REVIEW

File: packages/plugin-ext/package.json [hunk 1]
```diff
@@ -5,15 +5,16 @@
   "dependencies": {
     "@theia/core": "^0.3.8",
     "@theia/filesystem": "^0.3.8",
-    "@theia/workspace": "^0.3.8"
+    "@theia/workspace": "^0.3.8",
+    "ps-tree": "latest"
```
Comment (evidolob) [inline packages/plugin-ext/package.json]: Please use version that will be approved in eclipse CQ

Comment (mmorhun) [reply packages/plugin-ext/package.json]: ok

**Comment by mmorhun** [comment]
CQ: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16197

**Review by benoitf — APPROVED** [review]

**Review by evidolob — APPROVED** [review]

**Comment by benoitf** [comment]
@mmorhun do we need type B for this CQ as it will be part of Theia which is type A I think ?

**Comment by benoitf** [comment]
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16197

seems we can check in for now

**Pull request merged**